VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participants-panel/participants-panel.component.ts

```diff
@@ -621,22 +646,34 @@ export class ParticipantsPanelComponent implements OnInit, OnDestroy {
             ? false
             : true;
     }
-
-    private setParticipants() {
+    private updateParticipants() {
         const combined = [...this.nonEndpointParticipants, ...this.endpointParticipants];
-        combined.forEach(participant => {
-            const currentParticipant = this.participants.find(r => r.id === participant.id);
+        combined.forEach(item => {
+            const currentParticipant = this.participants.find(r => r.id === item.id);
             if (currentParticipant) {
-                if (currentParticipant instanceof LinkedParticipantPanelModel) {
-                    currentParticipant.participants.forEach(linkedParticpant => {
-                        this.updateParticipant(participant, linkedParticpant);
-                    });
-                } else {
-                    this.updateParticipant(participant, currentParticipant);
+                currentParticipant.updateParticipant(
+                    currentParticipant?.isMicRemoteMuted(),
+                    currentParticipant?.hasHandRaised(),
+                    currentParticipant?.hasSpotlight(),
+                    currentParticipant?.id,
+                    currentParticipant?.isLocalMicMuted(),
+                    currentParticipant?.isLocalCameraOff()
```

Comment:
Can this call this.updateParticipant same as setParticipants?

Reply:
removed